music-library/src/controllers/artist.js

```diff
@@ -0,0 +1,89 @@
+const db = require('../db/index');
+
+const createArtist = async (req, res) => {
+const { name, genre } = req.body
+
+  try {
+    const { rows: [ artist ] } = await db.query(`INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`, [name, genre])
```

Comment:
Formatting again here, would highly recommend running prettier across the whole project to take the heavy lifting out of it for you
```
const { name, genre } = req.body

try {
  const { rows: [ artist ] } = await db.query(`INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *`, [name, genre])
```